README.md

```diff
@@ -0,0 +1,78 @@
+Docker-based integration tests
+=====
+[![PyPI version](https://img.shields.io/pypi/v/pytest-docker?color=blue)](https://pypi.org/project/pytest-docker/)
+[![Python versions](https://img.shields.io/pypi/pyversions/pytest-docker)](https://pypi.org/project/pytest-docker/)
+[![Code style](https://img.shields.io/badge/formatted%20with-black-black)](https://github.com/psf/black)
+
+
+# Description
+Simple [pytest](http://doc.pytest.org/) fixtures that help you write integration
+tests with Docker and [docker-compose](https://docs.docker.com/compose/).
+Specify all necessary containers in a `docker-compose.yml` file and and
+`pytest-docker` will spin them up for the duration of your tests.
+
+This package is tested with Python versions `3.5`, `3.6`, `3.7` and `3.8` and
+`pytest` version 5. Python 2 is not supported.
+
+`pytest-docker` was originally created by André Caron.
+
+# Usage
+Here is a basic example a test that depends on a service that
+responds over HTTP:
+
+```python
+import pytest
+import requests
+
+from requests.exceptions import ConnectionError
+
+
+def is_responsive(url):
+    try:
+        response = requests.get(url)
+        if response.status_code == 200:
+            return True
+    except ConnectionError:
+        return False
+
+
+@pytest.fixture(scope="session")
+def http_service(docker_ip, docker_services):
+    """Ensure that HTTP service is up and responsive."""
+
+    port = docker_services.port_for("abc", 123)
```

Comment:
For this to be understandable, I feel we should also include a corresponding example `docker-compose.yml`, to illustrate what the `port_for` method does.

Or/also maybe include a full doc on the fixtures provided by the library and methods available on them?